ibc-core/ics02-client/types/src/error.rs

```diff
@@ -20,10 +19,19 @@ pub enum ClientError {
     ClientFrozen { description: String },
     /// client is not active. Status=`{status}`
     ClientNotActive { status: Status },
+    /// client is active when it should not be. Status=`{status}`
+    ClientNotInactive { status: Status },
     /// client state not found: `{client_id}`
     ClientStateNotFound { client_id: ClientId },
     /// client state already exists: `{client_id}`
     ClientStateAlreadyExists { client_id: ClientId },
+    /// Substitute client height `{substitute_height}` must be greater than subject client height `{subject_height}` during client recovery
```

Comment:
I see you use `should` and `must` in your error message. IMO, it's redundant. Can we just describe what happened? eg.

```
Substitute client height `{substitute_height}` is not greater than subject client height `{subject_height}` during client recovery
```